src/main/java/control/CreationModalViewController.java

```diff
@@ -272,14 +275,14 @@ private void createDirectorShot(MouseEvent event) {
             DirectorTimelineController directorTimelineController =
                     this.controllerManager.getDirectorTimelineControl();
 
-            directorTimelineController.addDirectorShot(
+            directorTimelineController.addDirectorShot(new DirectorShot(new GeneralShotData(
```

Comment:
for readability it might be better to define this Directorshot and/or generalshotdata on a seperate line before it is passed to addDirectorShot